src/plugins/timeConductor/ConductorAxis.vue

```diff
@@ -55,20 +57,69 @@ export default {
     }
   },
   emits: ['pan-axis', 'end-pan', 'zoom-axis', 'end-zoom'],
+  setup() {
+    const axisHolder = ref(null);
+    const { size: containerSize, startObserving } = useResizeObserver();
+
+    onMounted(() => {
+      startObserving(axisHolder.value);
+    });
+
+    return {
+      axisHolder,
+      containerSize
+    };
+  },
   data() {
     return {
       inPanMode: false,
       dragStartX: undefined,
       dragX: undefined,
-      zoomStyle: {}
+      zoomStyle: {},
+      rect: undefined
     };
   },
   computed: {
     inZoomMode() {
       return !this.inPanMode;
+    },
+    left() {
+      return this.rect.left;
```

Comment:
interesting, all we need to do is pan/drag in cardinal?

Reply:
not sure what you mean?

Reply:
to add the missing testcoverage identified in the codecov warning. We'd need to a test which pans in cardinal directions

Reply:
ah